### PR TITLE
feat: add docker image and docker compose

### PR DESCRIPTION
**Pull request by ImmanuelSegol**
@michael1011 I'm having an issue with docker-compose it boots `boltz-backend` and `boltz-middleware` instantly but won't wait for `boltz-regtest` to set up first,

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      - "30002" 
+      - "10009" 
+      - "10010" 
+      - "11009" 
+      - "11010" 
+      - "8081"
+    command: entrypoint.sh
+
+  boltz-backend:
+    image: 'moniasegol/test-boltz-backend:latest'
```
Comment (ImmanuelSegol) [inline docker/docker-compose.yml]: please upload to Boltz official docker hub

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    expose:
+      - "9000"
+    links:
+      - regtest
+    depends_on:
+      - regtest
+    command: npm start
+
+  boltz-middleware:
+    image: 'moniasegol/test-boltz-middleware:latest'
```
Comment (ImmanuelSegol) [inline docker/docker-compose.yml]: same as https://github.com/BoltzExchange/boltz-frontend/pull/161/files#r275119913

**Comment by michael1011** [comment]
Wait a second. Why would we want to have the frontend in a Docker image @ImmanuelSegol

**Comment by ImmanuelSegol** [comment]
> Wait a second. Why would we want to have the frontend in a Docker image @ImmanuelSegol

It will allow someone to easily deploy the application, perhaps someone wants to test it out without downloading all the npm modules, automated build, etc...

**Pull request closed without merging**